### DISCUSSION TITLE
email template for active new user

### DISCUSSION THREAD

**Discussion by TriumX** [Leantime/leantime#1628] [Q&A] [answered]
hello everyone,

i'm new here and not really understood about leantime. i'm trying to create new system for contract reminder to my boss using leantime 2.3.16. 1st thing i wanna ask is about how can i change email template for active users after invitation been send. can i change the template? and also can i change the main dashboard to  some other thing such as graph or some other thing?

thanks in advance.

**Comment by marcelfolaron** [comment, marked as answer]
Hi, the email template is currently hard coded into core/class.mailer.php 
In there you can adjust the email format.

**Reply by TriumX** [reply]
thanks for info. btw can i change main dashboard display? if i wanna put some graf after person login into the system.
